packages/account/src/Components/api-token/api-token-table-row-token-cell.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon, Text, Popover } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import ApiTokenClipboard from './api-token-clipboard.jsx';
+
+const ApiTokenTableRowTokenCell = ({ token, scopes }) => {
+    const [should_show_token, setShouldShowToken] = React.useState(false);
+
+    const HiddenPasswordDots = () => (
```

Comment:
Since `HiddenPasswordDots` is not dependent on `ApiTokenTableRowTokenCell`, can we move it outside the body of ApiTokenTableRowTokenCell? This will prevent remounting of `HiddenPasswordDots` whenever ApiTokenTableRowTokenCell Component re-renders

Reply:
sure, done :+1: